packages/hfs/src/proccessFiles.ts

```diff
@@ -0,0 +1,57 @@
+import { map, toArray } from 'rxjs/operators';
+import { transformSync } from '@babel/core';
+import babelRemoveImportsGetMeta from './babelRemoveImportsGetMeta';
+import md5 from 'md5';
+
+const tmpl = (arg, code) => `export default (obj) => {
+    ${arg !== '{}' ? 'const ' + arg + ' = obj;' : ''}
```

Comment:
can args be undefined?
```suggestion
    ${arg !== '{}' ? 'const ' + arg + ' = obj;' : ''}
```